### DISCUSSION TITLE
Core: Add position to variant options and values

### DISCUSSION THREAD

**Discussion by poppabear8883** [lunarphp/lunar#182] [Feature Requests]
Simply what the title suggests. I think it would be nice to have a manageable position for variants in order to have more options while building out the front-end.

**Comment by nicolalazzaro** [comment]
Hi @poppabear8883! 
Yes, this can be useful.

Expanding on your request I would add sorting for the following entities: ProductOption, ProductOptionValue and finally the relationships in ProductVariant.
This way there is a default sorting and if set directly on the variant a specific customization.

Perhaps the sorting on the ProductOption entity can be related to the ProductType entity. This way, the options are organised by product type to allow for clearer sorting.

What do you think about it?

**Reply by glennjacobs** [reply]
I agree, I think it's needed.
